.github/dependabot.yml

```diff
@@ -37,3 +40,13 @@ updates:
     labels:
       - "theme/dependencies"
       - "theme/website"
+  - package-ecosystem: github-actions
+    open-pull-requests-limit: 5
+    directory: /
+    labels:
+      - "theme/dependencies"
+      - "theme/ci"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+      time: "09:00"
```

Comment:
Generally speaking we prefer PRs to be atomic. Changing the dependabot rules is a different body of work than tuning the security scanner. Not a huge deal here but just for future reference.

Reply:
Good point, atomic PRs makes it easier to revert too, ✍️ noted it for the future